manual/manual/tutorials/coreexamples.etex

```diff
@@ -373,8 +383,24 @@ g r;;
 \pdfsection{Exceptions}
 
 OCaml provides exceptions for signalling and handling exceptional
-conditions. Exceptions can also be used as a general-purpose non-local
-control structure. Exceptions are declared with the "exception"
+conditions.
```

Comment:
I believe the clear consensus among Java developers is that exceptions should be used for truly exceptional conditions and not as a non-local go to or an alternate way of returning a value.  AFAIK this is not at all a controversial opinion.  But perhaps this doesn't carry over to OCaml, or the consensus among OCaml developers is different.

I could say something like this (or just drop the change):  "Exceptions can also be used as a general-purpose non-local control structure, although this should not be overused since it can make the code harder to understand."

Reply:
> the consensus among OCaml developers is different.

I don't think that there is a clear consensus on this subject to be honest.

> "Exceptions can also be used as a general-purpose non-local control structure, although this should not be overused since it can make the code harder to understand."

This sounds good.